source/agora/cli/client/Result.d

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 
-    The Agora client Result Constans
+    The Agora client Result Constant
```

Comment:
```suggestion
    Constants returned by the agora CLI client
```

Reply:
Although honestly I'd rather get rid of this module, I don't think a module with 3 `int` constant makes sense.